docker/buoy/Dockerfile

```diff
@@ -1,65 +1,31 @@
 FROM ros:galactic-ros-base
 ENV DEBIAN_FRONTEND=noninteractive
 
-# Useful tools
+# Necessary tools
 RUN apt update \
  && apt install -y \
-        apt-utils \
```

Comment:
Without apt-utils, I get a warning:
`debconf: delaying package configuration, since apt-utils is not installed
`

Reply:
Yeah I noticed that too. Didn't seem to cause any issues, but we can add it back to remove the red warning 👍🏽